src/makers/win32/appx.js

```diff
@@ -65,7 +65,10 @@ export default async (dir, appName, targetArch, forgeConfig, packageJSON) => { /
     opts.devCert = await createDefaultCertificate(opts.publisher, outPath);
   }
 
-  if (!opts.publisher.match(/^CN=/)) {
+  if (typeof opts.publisher.match !== 'function') {
```

Comment:
I agree with @malept 

The exact type checking is infinitely more readable and understandable than checking for a "match" property being a function